content/en/database_monitoring/setup_sql_server/selfhosted.md

```diff
@@ -0,0 +1,195 @@
+
+  See the [example check configuration][7] for a comprehensive description of all options, including how to use custom SQL queries to create your own metrics.
+
+  **Note**: The (default) provider `SQLOLEDB` is being deprecated. To use the newer `MSOLEDBSQL` provider, set the `adoprovider` variable to `MSOLEDBSQL` in your `sqlserver.d/conf.yaml` file after having downloaded the new provider from [Microsoft][9]. It is also possible to use the Windows Authentication and not specify the username/password with:
+
+  ```yaml
+  connection_string: "Trusted_Connection=yes"
+  ```
+
+2. [Restart the Agent][10].
+
+
+### Collecting logs (optional)
+
+1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+    ```yaml
+    logs_enabled: true
+    ```
+
+2. Add this configuration block to your `sqlserver.d/conf.yaml` file to start collecting your SQL Server logs:
+
+    ```yaml
+    logs:
+      - type: file
+        encoding: utf-16-le
+        path: "<LOG_FILE_PATH>"
+        source: sqlserver
+        service: "<SERVICE_NAME>"
+    ```
+
+    Change the `path` parameter value to be the file path of your SQL Server log file.
+
+    The `service` parameter allows you to tie your logs to other telemetry through a common `service` tag. To see how `service` is used throughout Datadog, see the documentation on [unified service tagging][8].
+
+3. [Restart the Agent][10].
+
+
+### Validate
+
+[Run the Agent's status subcommand][11] and look for `sqlserver` under the Checks section. Or visit the [Databases][12] page to get started!
+
+## Troubleshooting
+
+If you have installed and configured the integrations and Agent as described and it is not working as expected, see [Troubleshooting][13]
+
+## Further reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+
+[1]: /agent/basic_agent_usage#agent-overhead
+[2]: /database_monitoring/data_collected/#sensitive-information
+[3]: https://docs.microsoft.com/en-us/sql/tools/configuration-manager/tcp-ip-properties-ip-addresses-tab
+[4]: https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-2017
+[5]: https://docs.microsoft.com/en-us/sql/t-sql/statements/grant-server-permissions-transact-sql?view=sql-server-ver15
+[6]: https://app.datadoghq.com/account/settings#agent
+[7]: https://github.com/DataDog/integrations-core/blob/master/sqlserver/datadog_checks/sqlserver/data/conf.yaml.example
+[8]: /getting_started/tagging/unified_service_tagging
+[9]: https://docs.microsoft.com/en-us/sql/connect/oledb/oledb-driver-for-sql-server?view=sql-server-2017
+[10]: /agent/guide/agent-commands/#start-stop-and-restart-the-agent
+[11]: /agent/guide/agent-commands/#agent-status-and-information
+[12]: https://app.datadoghq.com/databases
+[13]: /database_monitoring/troubleshooting/?tab=sqlserver
```

Comment:
There isn't a sqlserver tab on this page yet, is that coming in another PR?

Reply:
This will be added sometime in the future. Can we leave it like this so we don't forget to update it later? It should not break anything.